SubmittingPatches-backports.rst

```diff
@@ -0,0 +1,344 @@
+======================================
+Submitting Patches to Ceph - backports
+======================================
+
+Most likely you're reading this because you intend to submit a GitHub pull
+request ("PR") targeting one of the stable branches ("nautilus", etc.) at
+https://github.com/ceph/ceph.
+
+Before you open that PR, please read (at the very least) the following general
+rules::
+
+Backports should state (either in the backport PR or in the tracker issue):
+
+1. what bug is fixed
+2. why this fix is the minimal way to do it
+3. why does this need to be fixed in <release>
+
+The following rules apply to the actual implementation of the backport:
+
+* all bugfixes and features should land in master first
+* commits to stable branches should be cherry-picked from master
+* cherry-picks must be done using ``git cherry-pick -x``
+* if a commit cannot be cherry-picked from master, the commit message must explain why that was not possible
```

Comment:
s/was/is/

Reply:
I intended for this sentence to be in past tense, since the commit message is written *after* the cherry-picking is done. But you're right - it's not consistent. I'll do `s/cannot/could not/`